internal/impl/kafka/enterprise/redpanda_migrator_bundle_output.tmpl.yaml

```diff
@@ -33,24 +33,33 @@ fields:
 mapping: |
   #!blobl
 
-  let redpandaMigratorOffsets = this.redpanda_migrator.with("seed_brokers", "consumer_group", "client_id", "rack_id", "max_message_bytes", "broker_write_max_bytes", "tls", "sasl")
-
-  let redpanda_migrator = this.redpanda_migrator.assign(
+  if ["topic", "key", "partition", "partitioner", "timestamp"].any(f -> this.redpanda_migrator.keys().contains(f)) {
+    root = throw("The topic, key, partition, partitioner and timestamp fields of the redpanda_migrator output must be left empty")
+  }
+  let rpMigratorMaxInFlight = this.redpanda_migrator.max_in_flight.or(1)
+  let redpandaMigrator = this.redpanda_migrator.assign(
     {
       "topic": "${! metadata(\"kafka_topic\").or(throw(\"missing kafka_topic metadata\")) }",
       "key": "${! metadata(\"kafka_key\") }",
       "partition": "${! metadata(\"kafka_partition\").or(throw(\"missing kafka_partition metadata\")) }",
       "partitioner": "manual",
-      "timestamp": "${! metadata(\"kafka_timestamp_unix\").or(timestamp_unix()) }"
+      "timestamp": "${! metadata(\"kafka_timestamp_unix\").or(timestamp_unix()) }",
+      "max_in_flight": $rpMigratorMaxInFlight
     }
   )
 
+  let redpandaMigratorOffsets = this.redpanda_migrator.with("seed_brokers", "consumer_group", "client_id", "rack_id", "max_message_bytes", "broker_write_max_bytes", "tls", "sasl")
+
   if this.schema_registry.keys().contains("subject") {
     root = throw("The subject field of the schema_registry output must not be set")
   }
-  let schema_registry = if this.schema_registry.length() > 0 { this.schema_registry.assign({"subject": "${! @schema_registry_subject }"}) }
-  if ["topic", "key", "partition", "partitioner", "timestamp"].any(f -> this.redpanda_migrator.keys().contains(f)) {
-    root = throw("The topic, key, partition, partitioner and timestamp fields of the redpanda_migrator output must be left empty")
+
+  let srMaxInFlight = this.schema_registry.max_in_flight.or(1)
+  let schemaRegistry = if this.schema_registry.length() > 0 {
+    this.schema_registry.assign({
+      "subject": "${! @schema_registry_subject }",
+      "max_in_flight": $srMaxInFlight
+    })
   }
```

Comment:
Just making sure that we set `max_in_flight` to `1` unless users set it to something else explicitly (and hopefully are aware they'll get out-of-order messages in that case).